doc/classes/SpriteFrames.xml

```diff
@@ -54,7 +54,7 @@
 			<argument index="0" name="anim" type="StringName">
 			</argument>
 			<description>
-				If [code]true[/code], the given animation will loop.
+   				Returns [code]true[/code] if the given animation is configured to loop when it ends playing, otherwise returns [code]false[/code].
```

Comment:
```suggestion
   				Returns [code]true[/code] if the given animation is configured to loop when it finishes playing. Otherwise, returns [code]false[/code].
```